node/core/runtime-api/src/lib.rs

```diff
@@ -302,33 +302,15 @@ where
 	let _timer = metrics.time_make_runtime_api_request();
 
 	macro_rules! query {
-		($req_variant:ident, $api_name:ident (), $sender:expr) => {{
+		($req_variant:ident, $api_name:ident ($($param:expr),*), $sender:expr) => {{
 			let sender = $sender;
 			let api = client.runtime_api();
-			let res = api.$api_name(&BlockId::Hash(relay_parent))
+			let res = api.$api_name(&BlockId::Hash(relay_parent), $( $param.clone() ),*)
```

Comment:
Perhaps this makes more sense?

```suggestion
			let res = api.$api_name(&BlockId::Hash(relay_parent) $(, $param.clone() )*)
```